src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs

```diff
@@ -5416,7 +5416,7 @@ static int M(string x, string y)
     }
 }
 ";
-            var verifier = CompileAndVerify(source, expectedOutput: "0123233", options: TestOptions.DebugExe);
+            var verifier = CompileAndVerify(source, expectedOutput: "0123233", options: TestOptions.DebugExe, parseOptions: TestOptions.RegularPreview);
```

Comment:
Why was the language version necessary? C# 9 should be the default at this point (we shouldn't be using "preview" until we add new features)

Reply:
not the case in 16.7 branch.